README.md

```diff
@@ -1,102 +1,95 @@
-# lint-prepush    
+# lint-prepush
+
 [![npm version](https://badge.fury.io/js/lint-prepush.svg)](https://www.npmjs.com/package/lint-prepush)
 [![npm downloads](https://img.shields.io/npm/dt/lint-prepush.svg)](https://www.npmtrends.com/lint-prepush)
 [![GitHub license](https://img.shields.io/github/license/theenadayalank/lint-prepush.svg)](https://github.com/theenadayalank/lint-prepush/blob/master/LICENSE)
 [![Build Status](https://travis-ci.org/theenadayalank/lint-prepush.svg?branch=master)](https://travis-ci.org/theenadayalank/lint-prepush)
 
+> Run linters on committed files of a GIT Branch
 
-> Run linters on committed files of a GIT Branch🔬
-
-## Getting Started 🔮
+## Getting Started
 
 This package will run linters on your project for the committed files in your branch.
 
-### Prerequisites🔭
-
-It require Node.js v6 or newer. It also requires a package to manage git hooks. I strongly suggest [Husky](https://github.com/typicode/husky) which I use for most of my projects.
-
-
-```js
-// package.json
-{
```

Comment:
I moved the `husky` configuration below where we are actually implementing `lint-prepush` and adjusting the Prerequisites list to be easier to scan.